docs/docs/getting-started/mini-constellation.md

```diff
@@ -0,0 +1,115 @@
+# Mini Constellation
+
+With `constellation mini`, users can quickly deploy and test Constellation without the need for a cloud subscription.
+
+The command uses virtualization to create a local cluster with one control-plane and one worker node.
+
+:::info
+
+Since mini Constellation is running on your local system, please note that common cloud features, such as load-balancing,
+attaching persistent storage, or auto-scaling, are unavailable.
+
+:::
+
+## Prerequisites
+
+* [Constellation CLI](./install.md#install-the-constellation-cli)
+* Linux operating system on amd64 hardware
```

Comment:
```suggestion
* Linux operating system on amd64 
```
Or similar but the distro also needs to be amd64. Not sure if people actually run i686 distros though anymore, they would already fail running the pre-built CLI.